x-pack/plugins/endpoint/server/routes/endpoints.ts

```diff
@@ -5,19 +5,44 @@
  */
 
 import { IRouter } from 'kibana/server';
-import { endpoints2 } from './endpoints_stubs';
+import { schema } from '@kbn/config-schema';
+
+export function registerEndpointsApi(router: IRouter) {
+  const endpointsHandler = async (context, req, res) => {
+    try {
+      if (!context.endpointPlugin) return res.internalError({ body: 'endpointPlugin is disabled' });
```

Comment:
This seems like it's likely not going to work--if I'm reading this correctly, if this endpoint is ever called, the `endpointPlugin` stuff _**will**_ be injected into the context, so you're not going to have a state where this is called and the `endpointPlugin` stuff doesn't exist on the context. That said, see comment about possible other patterns for DI.